decompiler/pipeline/controlflowanalysis/expression_simplification/rules/collapse_nested_constants.py

```diff
@@ -17,17 +18,17 @@ class CollapseNestedConstants(SimplificationRule):
     def apply(self, operation: Operation) -> list[tuple[Expression, Expression]]:
```

Comment:
Please add a doc string indicating which Exceptions can occur in this function.
Maybe this makes sense for the other functions in this class as well.

Reply:
Same as https://github.com/fkie-cad/dewolf/pull/352#discussion_r1371737160.